app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -881,6 +881,32 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
+    var useLocalFxAServer by booleanPreference(
```

Comment:
Maybe rename this to something like `allowDomesticChinaFxaServer`? This is a very clear name, to me at least. I'd also add a comment saying that this pref is only used in conjunction with Mozilla Online builds.

Word "local" could mean many different things in a broader context, and could easily be misunderstood for e.g. local development server.